package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "webm-maker-thing-idk",
```

Comment:
@OIRNOIR Do you want to have this name on here?

Reply:
But that name works fine